app/server/api-generator/src/main/java/io/syndesis/server/api/generator/swagger/UnifiedJsonDataShapeGenerator.java

```diff
@@ -288,6 +288,8 @@ private static DataShape unifiedJsonSchema(final String name, final String descr
         }
 
         final ObjectNode unifiedSchema = JsonSchemaHelper.newJsonObjectSchema();
+        // we're using Draft 4, in Draft 6+ this should be `$id`
```

Comment:
Can we move to Draft 6+ now?

Reply:
up to Atlasmap I think, and for another PR

Reply:
How about opening and issue in AltasMap for it and reference that here?

Reply:
Actually it us:

https://github.com/atlasmap/atlasmap/blob/70020a7f78feec388efb289352dc0593ef4eeee4/runtime/modules/json/core/src/main/java/io/atlasmap/json/inspect/SchemaInspector.java#L90

https://github.com/atlasmap/atlasmap/blob/70020a7f78feec388efb289352dc0593ef4eeee4/runtime/modules/json/core/src/test/resources/inspect/schema/flatprimitive-base-unrooted.json#L2

Reply:
Right, Mr. Smartypants

Reply:
Capt Obvious to you